plugins/processors/converter/converter.go

```diff
@@ -39,30 +40,32 @@ var sampleConfig = `
 `
 
 type Conversion struct {
-	Tag      []string `toml:"tag"`
-	String   []string `toml:"string"`
-	Integer  []string `toml:"integer"`
-	Unsigned []string `toml:"unsigned"`
-	Boolean  []string `toml:"boolean"`
-	Float    []string `toml:"float"`
+	Measurement []string `toml:"measurement"`
+	Tag         []string `toml:"tag"`
+	String      []string `toml:"string"`
+	Integer     []string `toml:"integer"`
+	Unsigned    []string `toml:"unsigned"`
+	Boolean     []string `toml:"boolean"`
+	Float       []string `toml:"float"`
 }
 
 type Converter struct {
-	Tags   *Conversion `toml:"tags"`
-	Fields *Conversion `toml:"fields"`
+	Tags   *Conversion     `toml:"tags"`
+	Fields *Conversion     `toml:"fields"`
+	Log    telegraf.Logger `toml:"-"`
```

Comment:
should log be private?

Reply:
It does needs to be exported, which is why we mark it unsettable from the toml.  This is due to how we inject the Logger from outside the package (which we have never been super happy with).  When adding this we also toyed around with injecting the logger on init: `Init(Logger) error` but ultimately decided not to inject it there.